pkg/apis/networking/v1alpha1/ingress_types.go

```diff
@@ -100,6 +100,11 @@ type IngressSpec struct {
 	// +optional
 	Rules []IngressRule `json:"rules,omitempty"`
 
+	// HTTPOption is the option of HTTP. It has the following three values:
+	// `HTTPOptionEnabled`, `HTTPOptionDisabled`, `HTTPOptionRedirected`
```

Comment:
Same here. We don't need HTTPOptionDisabled.

Reply:
Sounds good to me.

Reply:
We should just comment that right now disabled is not used.